agent/taskresource/volume/dockervolume.go

```diff
@@ -15,23 +15,41 @@ package volume
 
 import (
 	"github.com/aws/amazon-ecs-agent/agent/dockerclient/dockerapi"
+	"github.com/aws/amazon-ecs-agent/agent/taskresource"
 	"github.com/cihub/seelog"
+	"github.com/pkg/errors"
 
 	"encoding/json"
 	"sync"
 	"time"
 )
 
-// VolumeResource represents volume resource
+const (
+	//TaskScope indicates that the volume is created and deleted with task
```

Comment:
there needs to be a space between '//' and the comment. Please fix this everywhere